src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -538,12 +471,6 @@ namespace HVACMultiSpeedHeatPump {
         int i;                         // Index to speeds
         int j;                         // Index to speeds
         bool Found;                    // Flag to find autosize
-        int HeatingCoilInletNode;      // Heating coil inlet node number
-        int HeatingCoilOutletNode;     // Heating coil outlet node number
-        int CoolingCoilInletNode;      // Cooling coil inlet node number
-        int CoolingCoilOutletNode;     // Cooling coil outlet node number
-        int SuppHeatCoilInletNode;     // Supplemental heating coil inlet node number
-        int SuppHeatCoilOutletNode;    // Supplemental heating coil outlet node number
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:541]:(style),[variableScope],The scope of the variable 'HeatingCoilInletNode' can be reduced.
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:542]:(style),[variableScope],The scope of the variable 'HeatingCoilOutletNode' can be reduced.
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:543]:(style),[variableScope],The scope of the variable 'CoolingCoilInletNode' can be reduced.
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:544]:(style),[variableScope],The scope of the variable 'CoolingCoilOutletNode' can be reduced.
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:545]:(style),[variableScope],The scope of the variable 'SuppHeatCoilInletNode' can be reduced.
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:546]:(style),[variableScope],The scope of the variable 'SuppHeatCoilOutletNode' can be reduced.